tox.ini

```diff
@@ -33,6 +33,7 @@ commands =
     conda env create --prefix {envdir}/env --file requirements/horovod-cpu-environment.yml --force
     {envdir}/env/bin/python -m pip install horovod --no-cache-dir
     {envdir}/env/bin/horovodrun --check-build
+    {envdir}/env/bin/python -m pip install merlin-sok
```

Comment:
Is `merlin-sok` the correct package to install?

Reply:
No, let me revise the code.

Reply:
@edknv  I just realize the branch is on your repo, So I cannot modify it. the correct way to install sok is `pip install sparse_operation_kit`